exercises/hello_hapi/problem.fr.md

```diff
@@ -47,6 +47,8 @@ Pour que le serveur commence à écouter sur le port défini au préalable,
 appelez la fonction `start()` :
 
 ```js
-server.start();
+server.start(function () {
+  console.log('Server running at:', server.info.uri);
```

Comment:
Might need either fr language here for "Server running at:' or just have it empty.